UIDokan/Sources/UIDokan/Extentions/Images+Style.swift

```diff
@@ -0,0 +1,10 @@
+import UIKit.UIImage
+
+typealias Assets = UIImage
+
+// MARK: - Assets
+
+extension Assets {
+    static let accentColor = Assets(named: "AccentColor")!
+    static let comics22 = Assets(named: "comics22")!
+}
```

Comment:
We need only extension for UIImage.. So the expected result is 
```swift
extension UIImage { 
    static var comics22: UIImage { 
        UIImage(named: "comics22")!
    }
}

Reply:
Done